packages/relay/src/lib/eth.ts

```diff
@@ -1003,7 +1003,7 @@ export class EthImpl implements Eth {
       if (e instanceof JsonRpcError) {
         return e;
       }
-      return predefined.INTERNAL_ERROR();
+      return e.message ? predefined.INTERNAL_ERROR(e.message.toString()) : predefined.INTERNAL_ERROR();
```

Comment:
Q: why not just pass e into `predefined.INTERNAL_ERROR` and let it do this logic of if to pick up message?

Reply:
Fixed.